server/controllers/content/content.ts

```diff
@@ -21,6 +22,12 @@ export enum DOWNLOAD_STATUS {
     INDEXED = "DOWNLOADED",
     FAILED = "FAILED"
 }
+export enum CONTENT_UPDATE {
```

Comment:
This is wrong,  check the name of the enum and what it contains
and everything should be consistent in enum https://www.typescriptlang.org/docs/handbook/enums.html
which follows standard all the values should be in uppercase
Why this is required ?